common/openapi.yml

```diff
@@ -1011,14 +1011,16 @@ components:
       description: |
         Object containing metadata about the pipeline that was run, as well as pipeline run outputs if the run has completed successfully.
       type: object
-      required: [ pipelineName, pipelineVersion, toolVersion ]
+      required: [ pipelineName, pipelineVersion, toolVersion, userInputs ]
       properties:
         pipelineName:
           $ref: "#/components/schemas/PipelineName"
         pipelineVersion:
           $ref: "#/components/schemas/PipelineVersion"
         toolVersion:
           $ref: "#/components/schemas/PipelineToolVersion"
+        userInputs:
```

Comment:
does this being a required new response field make this a breaking change to the endpoints that use this component?

Reply:
it would be a breaking change for the CLI once the CLI uses it, right?

Reply:
im not sure, it _shouldnt_ care if something extra is provided in the response.  If it does there is probably a setting we can change so it doesnt

Reply:
ok yeah you're right. I got myself all confused about how our versioning works. don't mind me.